server/models/ConnectedAccount.js

```diff
@@ -43,6 +43,13 @@ export default (Sequelize, DataTypes) => {
     {
       paranoid: true,
 
+      indexes: [
```

Comment:
Is this enough to create the index in DB, no need to create a migration?

Reply:
Nice catch @Betree! It is not needed if you use `db.sync()`, since we don't, I'll add it to the migration.